README.md

```diff
@@ -40,23 +40,23 @@ Thu: Widespread haze between noon and 3pm. Sunny, high of 48. SSW wind 5-10 mph
 
 _"Closure" here includes any safety notices, detours, updates, or hazards posted by the trail authority. Currently only supports [the Pacific Crest Trail](https://www.pcta.org)._
 
-#### Number of closures in each region
+#### Number of closures in each region of a trail
 
 ##### Your message
 
-```closures```
+```closures pct```
 
 ##### Mouse's response
 
 ```text
 Closures by region: Southern California 5, Central California 0, Northern California 5, Oregon 1, Washington 3
```

Comment:
Why not shorten region names to SoCal, NorCal, etc.? It's pretty common practice and would make requests slightly easier to type.

Reply:
I like this idea a lot. We should be able to accept a range of region names; eg, the following pairs of invocations should be accepted and resolved the same:

- `closures pct norcal` and `closures pct Northern California`
- `closures pct washington` and `closures pct wa`

Probably this requires us to have one core/main name, and alias the others.

Ticketed to #17.